supplementary_style_guide/glossary_terms_conventions/product_conventions/red-hat-directory-server.adoc

```diff
@@ -0,0 +1,245 @@
+
+*Use it*: yes
+
+*Incorrect forms*: object class, objectclass
+
+*See also*: xref:object-class[object class]
+
+[discrete]
+[[red-hat-directory-server]]
+==== image:images/yes.png[yes] Red Hat Directory Server (noun)
+*Description*: Red Hat Directory Server (RHDS) is an LDAPv3-compliant directory server and the name of the product. Use the full product name in titles of guides. Outside of titles, refer to the product as Directory Server. Use the product name without an article. Do not use the acronym RHDS in documentation.
+
+*Use it*: yes
+
+*Incorrect forms*: RHDS
+
+*See also*: xref:directory-server[Directory Server]
+
+[discrete]
+[[replica]]
+==== image:images/yes.png[yes] replica (noun)
+*Description*: A replica is a copy of the Directory Server database on a different host. For example, a consumer can also be called a replica because it has a copy of the data received from the supplier.
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: 
+
+[discrete]
+[[slave]]
+==== image:images/no.png[no] slave (noun)
+*Description*: Do not use slave to refer to a consumer or hub.
+
+*Use it*: no
+
+*Incorrect forms*: 
+
+*See also*: xref:consumer[consumer], xref:hub[hub]
+
+[discrete]
+[[SSL]]
+==== image:images/no.png[no] SSL (noun)
+*Description*: The Secure Socket Layer (SSL) protocol is the insecure predecessor of TLS. Due to the weakness of the protocol, support of SSL has been removed in RHEL 8 and Directory Server 11. When you refer to TLS encryption or certificates, use TLS.
+
+*Use it*: no
+
+*Incorrect forms*: Secure Socket Layer, SSL
+
+*See also*: xref:tls[TLS]
+
+[discrete]
+[[starttls]]
+==== image:images/yes.png[yes] STARTTLS (noun)
+*Description*: When an LDAP client wants to use a TLS-encrypted connection after establishing a connection to the unencrypted LDAP port, the client sends the STARTTLS command.
+
+*Use it*: yes
+
+*Incorrect forms*: StartTLS, startTLS
+
+*See also*: xref:ldap[LDAP], xref:ldaps[LDAPS]
+
+[discrete]
+[[suffix]]
+==== image:images/yes.png[yes] suffix (noun)
+*Description*: The name of the entry at the top of the directory tree is called a suffix. In Directory Server, an instance can store multiple suffixes, and each suffix has its own database.
+
+*Use it*: yes
+
+*Incorrect forms*: 
+
+*See also*: 
+
+[discrete]
+[[supplier]]
+==== image:images/yes.png[yes] supplier (noun)
+*Description*: In an LDAP replication environment, suppliers send data to other servers.
+
+*Use it*: yes
+
+*Incorrect forms*: master
+
+*See also*: xref:consumer[consumer]
+
+[discrete]
+[[tls]]
+==== image:images/yes.png[yes] TLS (noun)
+*Description*: Transport Layer Security (TLS) is a cryptographic protocol for secure communication over networks. TLS is the successor of the insecure Secure Sockets Layer (SSL) protocol. RHEL 8 and 9, as well as Directory Server 11 and 12, only support TLS version 1.2 and later. Do not expand.
+
+*Use it*: yes
+
+*Incorrect forms*: SSL, SSL/TLS, TLS/SSL
+
+*See also*: 
```

Comment:
Since SSL links here, should TLS link back to it as well?

Reply:
Fixed